cli/cmd/plugin/unmanaged-cluster/main.go

```diff
@@ -19,6 +19,14 @@ var descriptor = cliv1alpha1.PluginDescriptor{
 	Aliases:     []string{"um", "uc", "unmanaged"},
 	Description: description,
 	Group:       cliv1alpha1.RunCmdGroup,
+
+	// TODO: jpmcb - when tanzu framework has bumped their version of controller-runtime,
+	// we can use the newest version of the PluginDescriptor from v0.10.0+ which uses the buildinfo.Version.
+	// Right now, since we are accessing the controller-runtime package, there is a build conflict
+	// with v0.7.0 of controller runtime (which tanzu-framework uses). See the following github issue:
+	// https://github.com/vmware-tanzu/tanzu-framework/issues/444
+	// For now, we can just use a raw string for this build instead of a dynamic buildtime env-var.
+	Version: "v0.10.0",
```

Comment:
I wonder if it would be better to use an ldflag to do something like `-X cli/cmd/plugin/unmanaged-cluster/main.PluginVersion=${BUILD_VERSION}` in the Makefile. Assuming the addition of a `PluginVersion` variable that would then be used here to set the `PluginDescriptor.Version` field. Then this will just be set automatically based on the build being done.

Reply:
That's definitely a possibility - we'd be introducing and owning some of that build time complexity, but I like that and I think would be much more maintainable over the long term.

I'll take a stab at it in this PR tomorrow